packages/office-ui-fabric-react/src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -74,8 +74,8 @@ export class Breadcrumb extends BaseComponent<IBreadcrumbProps, IBreadcrumbState
 
     return (
       <div className={ css('ms-Breadcrumb', className, styles.root) } ref='renderingArea'>
-        <FocusZone direction={ FocusZoneDirection.horizontal }>
-          <ul className={ css('ms-Breadcrumb-list', styles.list) }>
+        <FocusZone direction={ FocusZoneDirection.horizontal } role='navigation'>
+          <ol className={ css('ms-Breadcrumb-list', styles.list) }>
             { renderedOverflowItems && renderedOverflowItems.length ? (
               <li className={ css('ms-Breadcrumb-overflow', styles.overflow) } key={ OVERFLOW_KEY } ref={ OVERFLOW_KEY }>
                 <div className={ css('ms-Breadcrumb-overflowButton ms-Icon ms-Icon--More', styles.overflowButton) }
```

Comment:
Why isn't the overflow button a <button>? Also, it needs to have a title attribute like "Breadcrumb Overflow" so that it will read something when focused and will have a tooltip on hover